proto/teleport/lib/teleterm/v1/cluster.proto

```diff
@@ -34,7 +34,9 @@ message Cluster {
   // For leaf clusters, it has the form of /clusters/:rootClusterId/leaves/:leafClusterId where
   // leafClusterId is equal to the name property of the cluster.
   string uri = 1;
-  // name is used throughout the Teleport Connect codebase as the cluster name.
+  // name is the site name of Teleport cluster.
+  // Only present in root clusters if the user has logged in (the name comes from the key).
```

Comment:
Maybe "certificate" would me more clear here than "key" as it has fewer meanings in this context.